Can no longer compare arrays in version 0.5.6

## Version 
From 0.5.5 to 0.5.6

## Description
I was able to compare native arrays in version 0.5.5 but they no longer have the Comparable trait satisfied in 0.5.6


## Reproduction
```rust
use comparable::*;

fn main() {
    let t1 = [0u8; 5];
    let t2 = [0u8; 5];
    assert_changes!(&t1, &t2, Changed::Unchanged);
    print!("{:?}", t1.comparison(&t2));
}
```

This builds, run and works in 0.5.5

In 0.5.6 we have the following errors:
```
for assert_changes!(&t1... => error[E0599]: the method `comparison` exists for reference `&[u8; 5]`, but its trait bounds were not satisfied
for t1.comparison(&t2... => error[E0599]: no method named `comparison` found for array `[u8; 5]` in the current scope
```

## Expected behaviour
Same as in 0.5.5, or is there an easy workaround that does not mean using a vec instead of an array ?

--- a/comparable/src/lib.rs
+++ b/comparable/src/lib.rs
@@ -911,6 +911,8 @@
 //!
 //! Unions cannot derive [`Comparable`] instances at the present time.
 #[doc(hidden)]
+pub mod array;
+#[doc(hidden)]
 pub mod boxes;
 #[doc(hidden)]
 pub mod empty;
@@ -919,11 +921,15 @@ pub mod map;
 #[doc(hidden)]
 pub mod opt;
 #[doc(hidden)]
+pub mod path;
+#[doc(hidden)]
 pub mod scalar;
 #[doc(hidden)]
 pub mod set;
 #[doc(hidden)]
 pub mod string;
+#[doc(hidden)]
+pub mod tuple;
 #[doc(hidden)]
 pub mod types;
 
@@ -931,6 +937,8 @@ pub mod types;
 pub use crate::map::*;
 #[doc(hidden)]
 pub use crate::opt::*;
+#[doc(hidden)]
+pub use crate::path::*;
 #[doc(hidden)]
 pub use crate::scalar::*;
 #[doc(hidden)]

--- a/comparable/src/lib.rs.in
+++ b/comparable/src/lib.rs.in
@@ -1,4 +1,6 @@
 #[doc(hidden)]
+pub mod array;
+#[doc(hidden)]
 pub mod boxes;
 #[doc(hidden)]
 pub mod empty;
@@ -7,11 +9,15 @@ pub mod map;
 #[doc(hidden)]
 pub mod opt;
 #[doc(hidden)]
+pub mod path;
+#[doc(hidden)]
 pub mod scalar;
 #[doc(hidden)]
 pub mod set;
 #[doc(hidden)]
 pub mod string;
+#[doc(hidden)]
+pub mod tuple;
 #[doc(hidden)]
 pub mod types;
 
@@ -19,6 +25,8 @@ pub mod types;
 pub use crate::map::*;
 #[doc(hidden)]
 pub use crate::opt::*;
+#[doc(hidden)]
+pub use crate::path::*;
 #[doc(hidden)]
 pub use crate::scalar::*;
 #[doc(hidden)]

--- a/comparable_test/test/array.rs
+++ b/comparable_test/test/array.rs
@@ -1,0 +1,42 @@
+use comparable::{Changed::*, *};
+
+// Regression test for https://github.com/jwiegley/comparable/issues/13:
+// arrays lost their `Comparable` impl in 0.5.6 because `pub mod array;` was
+// dropped from the generated `lib.rs`.  These tests exercise the array impl so
+// the module stays wired in.
+
+// The exact reproduction from the issue report.
+#[test]
+fn test_array_issue_13_repro() {
+	let t1 = [0u8; 5];
+	let t2 = [0u8; 5];
+	assert_changes!(&t1, &t2, Unchanged);
+	assert_eq!(t1.comparison(&t2), Unchanged);
+}
+
+#[test]
+fn test_array_unchanged() {
+	let t1 = [1u8, 2, 3, 4, 5];
+	let t2 = [1u8, 2, 3, 4, 5];
+	assert_changes!(&t1, &t2, Unchanged);
+}
+
+#[test]
+fn test_array_single_change() {
+	let t1 = [1u8, 2, 3, 4, 5];
+	let t2 = [1u8, 9, 3, 4, 5];
+	assert_changes!(&t1, &t2, Changed([Unchanged, Changed(U8Change(2, 9)), Unchanged, Unchanged, Unchanged]));
+}
+
+#[test]
+fn test_array_multiple_changes() {
+	let t1 = [1u32, 2, 3];
+	let t2 = [4u32, 2, 6];
+	assert_changes!(&t1, &t2, Changed([Changed(U32Change(1, 4)), Unchanged, Changed(U32Change(3, 6))]));
+}
+
+#[test]
+fn test_array_describe() {
+	let t = [10u8, 20, 30];
+	assert_eq!(t.describe(), [10u8, 20, 30]);
+}

--- a/comparable_test/test/path.rs
+++ b/comparable_test/test/path.rs
@@ -1,0 +1,51 @@
+use comparable::{Changed::*, *};
+use std::path::{Path, PathBuf};
+
+// Regression test for https://github.com/jwiegley/comparable/issues/13:
+// `Path`/`PathBuf` lost their `Comparable` impl in 0.5.6 because `pub mod
+// path;` (and the `PathBufChange` re-export) was dropped from the generated
+// `lib.rs`.
+
+#[test]
+fn test_pathbuf_unchanged() {
+	let p1 = PathBuf::from("/foo/bar");
+	let p2 = PathBuf::from("/foo/bar");
+	assert_changes!(&p1, &p2, Unchanged);
+}
+
+#[test]
+fn test_pathbuf_changed() {
+	let p1 = PathBuf::from("/foo/bar");
+	let p2 = PathBuf::from("/foo/baz");
+	assert_changes!(&p1, &p2, Changed(PathBufChange(PathBuf::from("/foo/bar"), PathBuf::from("/foo/baz"))));
+}
+
+#[test]
+fn test_pathbuf_describe() {
+	let p = PathBuf::from("/foo/bar");
+	assert_eq!(p.describe(), PathBuf::from("/foo/bar"));
+}
+
+#[test]
+fn test_path_unchanged() {
+	let p1 = PathBuf::from("/foo/bar");
+	let p2 = PathBuf::from("/foo/bar");
+	assert_changes!(p1.as_path(), p2.as_path(), Unchanged);
+}
+
+#[test]
+fn test_path_changed() {
+	let p1 = PathBuf::from("/foo/bar");
+	let p2 = PathBuf::from("/foo/baz");
+	assert_changes!(
+		p1.as_path(),
+		p2.as_path(),
+		Changed(PathBufChange(PathBuf::from("/foo/bar"), PathBuf::from("/foo/baz")))
+	);
+}
+
+#[test]
+fn test_path_describe() {
+	let p: &Path = Path::new("/foo/bar");
+	assert_eq!(p.describe(), PathBuf::from("/foo/bar"));
+}

--- a/comparable_test/test/test.rs
+++ b/comparable_test/test/test.rs
@@ -3,13 +3,16 @@
 // #[comparable_ignore] and the various derive shapes.
 #![allow(dead_code)]
 
+mod array;
 mod boxes;
 mod empty;
 mod enums;
 mod map;
 mod opt;
+mod path;
 mod scalar;
 mod set;
 mod string;
 mod structs;
+mod tuple;
 mod unions;

--- a/comparable_test/test/tuple.rs
+++ b/comparable_test/test/tuple.rs
@@ -1,0 +1,31 @@
+use comparable::{Changed::*, *};
+
+// Regression test for https://github.com/jwiegley/comparable/issues/13:
+// tuples lost their `Comparable` impl in 0.5.6 because `pub mod tuple;` was
+// dropped from the generated `lib.rs`.
+
+#[test]
+fn test_tuple_pair_unchanged() {
+	assert_changes!(&(1u8, 2u32), &(1u8, 2u32), Unchanged);
+}
+
+#[test]
+fn test_tuple_pair_changed() {
+	assert_changes!(&(1u8, 2u32), &(1u8, 9u32), Changed((Unchanged, Changed(U32Change(2, 9)))));
+}
+
+#[test]
+fn test_tuple_pair_both_changed() {
+	assert_changes!(&(1u8, 2u32), &(4u8, 9u32), Changed((Changed(U8Change(1, 4)), Changed(U32Change(2, 9)))));
+}
+
+#[test]
+fn test_tuple_triple_changed() {
+	assert_changes!(&(1u8, 2u32, 3u64), &(1u8, 9u32, 3u64), Changed((Unchanged, Changed(U32Change(2, 9)), Unchanged)));
+}
+
+#[test]
+fn test_tuple_describe() {
+	let t = (1u8, 2u32);
+	assert_eq!(t.describe(), (1u8, 2u32));
+}

